website/pages/docs/concepts/storage.mdx

```diff
@@ -0,0 +1,74 @@
+
+Many other options for storage are available with community support - see our [Storage Configuration](/docs/configuration/storage) section for more information.
+
+## Backups & Restores
+
+Due to the highly flexible nature of Vault's potential storage configurations, providing exact guidance on backing up and restoring Vault is challenging.
+
+When backing up Vault, there are two pieces to consider:
+
+1. Vault's encrypted data in the storage backend
+2. Configuration files and management scripts for running the Vault server
+
+There's also a big question - what is the error case you're trying to guard against by saving a backup?
+
+### The Big Question - Why Take Backups?
+
+It's important to consider the question of "why take a backup" while developing your ongoing
+backup strategy.
+
+We do not recommend backups as protection against the failure of an individual machine or data
+center. Vault servers can run in clusters. Vault Enterprise supports replicated clusters for
+performance and disaster recovery. With open source features, a Vault cluster can extend across
+multiple availability zones within a region.
+
+As you develop a plan for backing up and restoring Vault, include clustering in your plan to
+address redundancy concerns.
+
+### Vault's Persisted Data
+
+Backups and restores are ideally performed while Vault is offline. If offline backups are not
+feasible, we recommend using a storage backend that supports atomic snapshots (such as
+[Consul](https://www.consul.io/docs/commands/snapshot.html) or [Integrated
+Storage](/docs/commands/operator/raft#snapshot)).
+
+**If your storage backend does not support atomic snapshots, we recommend only taking offline backups.**
+
+We recommend taking backups **before**, but not during, write operations to the `/sys` API
+(excluding the `/sys/leases`, `/sys/namespaces`, `/sys/tools`, `/sys/wrapping`, `/sys/policies`, and
+`/sys/pprof` endpoints). Some examples of workflows that write to the `/sys` API are upgrades,
+server moves, and rekeys. In the future, this guidance may change for the Integrated Storage
+backend.
+
+To perform a backup or restore of Vault's encrypted data when using a HashiCorp-supported storage
+backend, see the instructions linked below. For other storage backends, follow the documentation of
+that backend for taking and restoring backups.
+
+* Integrated Storage [snapshots](/docs/commands/operator/raft#snapshot)
+* Consul [snapshots](https://www.consul.io/docs/commands/snapshot.html)
+
+### Configuration
+
+In addition to backing up Vault's encrypted data via the storage backend, you may also wish to
+save the server configuration files, any scripts for managing the Vault service, and ensure you
+can reinstall any user-installed plugins. The location of these files will be specific to your
+installation of Vault.
+
+~> **NOTE**: Some of your configuration may be sensitive (a Vault token for Transit Autounseal
```

Comment:
Suggest we add something after this to the effect that snapshots are encrypted and thus do not share this concern.